backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TypeImpl.java

```diff
@@ -690,6 +729,31 @@ public boolean eIsSet(int featureID) {
         return super.eIsSet(featureID);
     }
 
+    @Override
+    public EList<Membership> visibleMemberships(EList<Namespace> excluded, boolean isRecursive, boolean includeAll) {
+        // Inherited members are visible members of a type
+        BasicEList<Type> excludedTypes = excluded.stream()
+                .filter(Type.class::isInstance)
+                .map(Type.class::cast)
+                .collect(toCollection(BasicEList<Type>::new));
+
+        EList<Membership> visibleMemberships = super.visibleMemberships(excluded, isRecursive, includeAll);
+        EList<Membership> inheritedMemberships = inheritedMemberships(excludedTypes);
+        List<Membership> data = Stream.concat(visibleMemberships.stream(),
+                inheritedMemberships.stream())
+                .filter(rel -> includeAll || rel.getVisibility() == VisibilityKind.PUBLIC)
+                .toList();
+        return new BasicEList<>(data);
+
```

Comment:
Please remove empty lines in this file and check others.

Reply:
Done

Reply:
donr